README.md

```diff
@@ -110,14 +117,39 @@ namespace Yoursite.Infrastructure.Environments
     }
 }
 ```
+In the example above it loads the setting/configuration value from the environment variables in DXP or IIS.  
+```csharp
+using Addon.Episerver.EnvironmentSynchronizer;
+using EPiServer.ServiceLocation;
+using Microsoft.Extensions.Configuration;
+
+namespace Yoursite.Infrastructure.Environments
+{
+   [ServiceConfiguration(typeof(IEnvironmentNameSource))]
+    public class EnvironmentNameSource : IEnvironmentNameSource
+    {
+        private readonly IConfiguration Configuration;
+
+        public EnvironmentNameSource(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public string GetCurrentEnvironmentName()
+        {
+            return Configuration["EnvironmentSettings.Environment"];
+        }
+    }
+}
+```
 In the example above it loads the setting/configuration value from appsettings.json with the name "EnvironmentSettings.Environment".  
 ```json
 {
   "EnvironmentSettings.Environment": "dev",
 }
 ```
-### DXP variable episerver:EnvironmentName support
-If you don´t implement the logic specified above. The DXP variable ´episerver:EnvironmentName´ will be used. More information about the DXP environments and the appsetting can be found on [https://world.episerver.com/documentation/developer-guides/digital-experience-platform/development-considerations/environment-configurations/](https://world.episerver.com/documentation/developer-guides/digital-experience-platform/development-considerations/environment-configurations/)  
+### DXP variable ASPNETCORE_ENVIRONMENT support
```

Comment:
Is this change related or just a fix that should be separated into another PR?

Reply:
Its a fix. Do you want me to put it in another PR?

Reply:
Yes, please.

Reply:
Can be found in a new PR